scripts/ci.sh

```diff
@@ -12,5 +18,45 @@ if [ ! "$HARDHAT_FORK_BLOCK" ]; then
   echo
 fi
 
+echo "Starting Hasura on port ${CI_HASURA_PORT}..."
+
+# comment out DOCKER_GATEWAY_HOST if running locally on macOS
+export DOCKER_GATEWAY_HOST=172.17.0.1
```

Comment:
can we put this in the .env instead?

Reply:
yes, it should also be put in the github environment